src/layout/AppLayout.tsx

```diff
@@ -1,23 +1,35 @@
+import localFont from 'next/font/local'
 import { PropsWithChildren } from 'react'
+import { BottomGrid } from 'src/components/nav/BottomGrid'
 import { Footer } from 'src/components/nav/Footer'
 import { Header } from 'src/components/nav/Header'
 import { InfoBanner } from 'src/components/nav/InfoBanner'
+import { TopBlur } from 'src/components/nav/TopBlur'
 import { PollingWorker } from 'src/features/polling/PollingWorker'
 import { HeadMeta } from 'src/layout/HeadMeta'
 
 interface Props {
   pathName: string
 }
 
+const foundersGrotesk = localFont({
+  src: '../../public/fonts/founders-grotesk-medium.woff2',
+  variable: '--font-founders-grotesk',
```

Comment:
This should be in `fonts.css` with the other font defs. Is there a reason it was defined here?